content/hazelcast.tex

```diff
@@ -0,0 +1,175 @@
+
+\chapter{Hazelcast}
+\chapterauthor{Vanessa Jörns, Tobias Schiffmann and Victor Veal}
+
+\section{Hazelcast Introduction}
+
+Hazelcast is a company that is developing an in-memory computing platform consisting of Hazelcast IMDG, Hazelcast Jet and Hazelcast Cloud. “Hazelcast Jet is an embeddable, distributed computing platform for fast processing of big data sets”. It is built on the foundation of Hazelcast IMDG on which this chapter focuses \cite{hazelcast}.\\
```

Comment:
There are lots of `\\` at the end of lines. This does not look like good LaTeX. There are only few good reasons to force line breaks. If you want to separate paragraphs, please use clear lines instead. In most other cases, LaTeX already does the right thing.

Reply:
This has not been resolved, either.